File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// JellyBellyWiki SDK exists test

require_once __DIR__ . '/../jellybellywiki_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = JellyBellyWikiSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# JellyBellyWiki SDK exists test

require "minitest/autorun"
require_relative "../JellyBellyWiki_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = JellyBellyWikiSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
